__tests__/app.test.js

```diff
@@ -59,4 +61,56 @@ describe('app', () => {
                 })
         })
     })
+
+    describe('/api/articles/:article_id', () => {
+        test("should respond with a json object of an article", () => {
+            return request(app)
+                .get("/api/articles/1")
+                .expect(200)
+                .then(({body}) => {
+                    const {article} = body;
+                    expect(article.article_id).toBe(1);
+                    expect(article).toHaveProperty('author', expect.any(String))
+                    expect(article).toHaveProperty('title', expect.any(String))
+                    expect(article).toHaveProperty('article_id', expect.any(Number))
```

Comment:
Would you expect any number here? Be more specific on which article id you are expecting to receive

Reply:
....just seen line 72 which would make this line redundant anyway 👍